openapi/openapi.json

```diff
@@ -27473,9 +27473,16 @@
       "type": "object",
       "required": [
         "managementState",
-        "forceRedeploymentReason"
+        "forceRedeploymentReason",
+        "backendQuotaGiB"
       ],
       "properties": {
+        "backendQuotaGiB": {
+          "description": "BackendQuotaGiB sets the etcd backend storage size limit in gibibytes. Defaults to 8GiB.",
+          "type": "integer",
+          "format": "int64",
+          "default": 0
```

Comment:
shouldn't that be 8? :)

Reply:
My bad sry 😔

Reply:
actually the tags are correct
```
// +kubebuilder:default=8
```

i dont know why the generated is wrong !!!

Reply:
I regenerated few times 

i have this tag 

```
// +kubebuilder:default=8
```

still the same, maybe it is the `int64` default ?